.env.olympia

```diff
@@ -0,0 +1,12 @@
+# This is a special configuration file for running code-manager locally using a
+# local addons-server instance.
+
+REACT_APP_API_HOST=http://olympia.test
+
+# The default port (`3000`) is taken by addons-frontend when we run the whole
+# addons-server stack.
+PORT=4000
```

Comment:
What about using 4001 so it doesn't conflict with a prod build? This could be helpful for cookies and other domain related things in the browser.

Reply:
Good idea!